### PR TITLE
Add etherscan official links scraper

### PR DESCRIPTION
**Pull request by LucoLe**
#### Summary

Adds official links to the etherscan scraper.
Fixes the issue with showing the blog link on the project's page.

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: lib/sanbase/external_services/etherscan/scraper.ex [hunk 1]
```diff
@@ -65,37 +65,77 @@ defmodule Sanbase.ExternalServices.Etherscan.Scraper do
...
+    |> Floki.attribute("href")
+    |> List.first()
+  end
+
+  defp email(nil) do
+    nil
+  end
+
+  defp email(mailto_string) do
+    String.slice(mailto_string, 7..-1)
```
Comment (IvanIvanoff) [inline lib/sanbase/external_services/etherscan/scraper.ex]: Can you add a comment why this is `slice 7..-1`?

Comment (IvanIvanoff) [reply lib/sanbase/external_services/etherscan/scraper.ex]: Better way to do this is with pattern matching in the function: `defp email("mailto:" <> email), do: email`.
Here is an example output from `iex`:
```
iex(1)> str = "mailto:ivan@gmail.com"
"mailto:ivan@gmail.com"
iex(2)> "mailto:" <> email = str
"mailto:ivan@gmail.com"
iex(3)> email
"ivan@gmail.com"
```

File: app/src/pages/Detailed/DetailedGQL.js [hunk 2]
```diff
@@ -10,6 +10,10 @@ export const projectBySlugGQL = gql`
       ticker,
       description,
       websiteLink,
+      email,
+      bitcointalkLink,
```
Comment (IvanIvanoff) [inline app/src/pages/Detailed/DetailedGQL.js]: where is this used?

Comment (LucoLe) [reply app/src/pages/Detailed/DetailedGQL.js]: This is not used anywhere at the moment but as agreed with @valo we should scrape the page for any official link that could be found.

File: lib/sanbase/external_services/etherscan/scraper.ex [hunk 3]
```diff
@@ -65,37 +65,78 @@ defmodule Sanbase.ExternalServices.Etherscan.Scraper do
...
+    |> List.first()
+  end
+
+  defp email(nil) do
+    nil
+  end
+
+  defp email(mailto_string) do
+    # Removes the mailto part of the string "mailto:eos@block.one"
+    String.slice(mailto_string, 7..-1)
```
[the remark this answers is not in the document]
Comment (LucoLe) [reply lib/sanbase/external_services/etherscan/scraper.ex]: nice one, thanks

**Pull request merged**